configs/setups/awiesm/awiesm.yaml

```diff
@@ -338,28 +379,36 @@ fesom:
             "*":
                 is: invalid
 
+        # use_ciso switch
+        choose_recom.use_ciso:
+                true:
+                        add_config_sources: 
+                                io: "${namelist_dir}/namelist.io.recom.ciso"
+                                oce: "${namelist_dir}/namelist.oce.recom.ciso"
+                        num_tracers: 33 
 #########################################################################################
 
 
 recom:
-        choose_general.version:
-                2.1-recom:
+        choose_general.with_recom:
+                True:
                         version: "2.0"
-                        bgc_num: 31
-                        benthos_num: 8
-                        bottflx_num: 8
-                        add_namelist_changes:
-                            namelist.recom:
-                                pavariables:
-                                    bgc_num: "${bgc_num}"
-                                    benthos_num: "${benthos_num}"
-                                    bottflx_num: "${bottflx_num}"
-
-                        data_path: /work/ollie/mbutzin/fesom2/input/dust/
+#                        bgc_num: 31
+#                        benthos_num: 8
+#                        bottflx_num: 8
+#                        add_namelist_changes:
+#                            namelist.recom:
+#                                pavariables:
+#                                    bgc_num: "${bgc_num}"
+#                                    benthos_num: "${benthos_num}"
+#                                    bottflx_num: "${bottflx_num}"
+
+                        data_path: /home/a/a270105/initial_files/pi_init/dust_coremesh/
```

Comment:
Are we okay with deleting all of this for previous versions of AWIESM-2.1-Recom?

Also, we have here 2 datapaths for 2 different computers. I think we need a `choose_computer.name` here right?